pkgs/os-specific/linux/dracut/default.nix

```diff
@@ -0,0 +1,100 @@
+{ stdenv
+, lib
+, fetchurl
+, makeWrapper
+, pkg-config
+, kmod
+, asciidoc
+, libxslt
+, docbook5
+, docbook_xsl
+, docbook_xsl_ns
+, coreutils
+, utillinux
+, gnused
+, gnugrep
+, squashfsTools
+, cpio
+, binutils
+, gzip
+, bzip2
+, lz4
+, lzop
+, zstd
+, xz
+}:
+
+stdenv.mkDerivation rec {
+  name = "dracut";
+  version = "049";
+  src = fetchurl {
```

Comment:
Is there a reason why you didn't use fetchFromGitHub?

Reply:
Does that fetch the releases?